### DISCUSSION TITLE
unable to login my github email account

### DISCUSSION THREAD

**Discussion by jackylin516888** [community/community#123121] [Code Security]
### Select Topic Area

General

### Body

hi there, I was unable to login my github email account jacklin5168@qq.com, please help address it. Thanks

**Comment by techpond31** [comment]
👋 If you're having trouble signing in to your GitHub account with the email address jacklin5168@qq.com, try these troubleshooting steps:

1. Open the GitHub login page in your browser: https://github.com/login
2. Click on the "Need help?" link beneath the password input field.
3. Choose "I can't sign in".
4. Select the option "I forgot my password" and fill in the email address jacklin5168@qq.com to initiate the password reset process.
5. Check your QQ Mail inbox for the password reset email. If you can't find the email, make sure to check your junk or spam folders.
6. Follow the instructions in the email to reset your password.

If you still can't sign in after attempting these steps, consider the following:

- Make sure that you entered the correct email address. Typos or misspellings can occur, so double-check the address carefully.
- If you signed up using Google Single Sign-On (SSO), Facebook SSO, or another SSO method, ensure that you're trying to sign in with the same method you originally registered. Otherwise, sign in using the SSO method to avoid entering incorrect credentials.
- If you changed your email address or transferred ownership of the account, double-check that you're using the correct email address tied to the account.
- If you suspect someone gained unauthorized access to your account, contact GitHub Support immediately. They can help secure your account and prevent unauthorized access.

Remember to never share your password with anyone, regardless of whom they claim to represent. Always use strong, unique passwords and consider enabling Multi-Factor Authentication (MFA) to add an extra layer of security to your GitHub account.
